### ISSUE TITLE
test_workspaces/python3_grpc:test fails with bazel 6.0

### ISSUE THREAD

**Issue by kevinchen-verkada** [rules-proto-grpc/rules_proto_grpc#223]
### Issue Description

I cloned the repository to my local machine(macbook) and go into `rules_proto_grpc/test_workspaces/python3_grpc` and ran `bazel test :all`.

I think we need to upgrade [GRPC_VERSION](https://github.com/rules-proto-grpc/rules_proto_grpc/blob/58f568397f3707e072c3e94493c380edea04337c/repositories.bzl#L10) to at least 1.49.0 to include the fix from `upb`. I verified it in my own repo.

### Log Output

```plain text
ERROR: 
/external/bazel_tools/platforms/BUILD:89:6: in alias rule @bazel_tools//platforms:windows: Constraints from @bazel_tools//platforms have been removed. Please use constraints from @platforms repository embedded in Bazel, or preferably declare dependency on https://github.com/bazelbuild/platforms. See https://github.com/bazelbuild/bazel/issues/8622 for details.
ERROR: 
/external/bazel_tools/platforms/BUILD:89:6: Analysis of target '@bazel_tools//platforms:windows' failed
ERROR: 
/external/upb/BUILD:257:11: errors encountered resolving select() keys for @upb//:textformat
ERROR: Analysis of target '//:py_lib' failed; build aborted:
```

### rules_proto_grpc Version

Ran the test target on the master branch at the time of filing this bug

### Bazel Version

6.0.0-pre.20220922.1

### OS

macos/intel

**Comment by aaliddell** [comment]
Bazel 6 isn't technically supported yet, since that's still prerelease, but the grpc version will be bumped in 4.3.0 which will add 6 support.

**Comment by aaliddell** [comment]
This was closed by 4.3.0, which picked up the newer grpc version
